src/EnergyPlus/WindowAC.cc

```diff
@@ -1499,6 +1499,36 @@ namespace WindowAC {
         } // WindAC(WindACNum)%DXCoilType_Num == CoilDX_CoolingHXAssisted && *
     }
 
+    bool GetWindowACNodeNumber(EnergyPlusData &state, int const NodeNumber)
+    {
+        if (state.dataWindowAC->GetWindowACInputFlag) {
+            GetWindowAC(state);
+            state.dataWindowAC->GetWindowACInputFlag = false;
+        }
+
+        bool windowACOutdoorAir = false;
+
+        for (int windowACIndex = 1; windowACIndex <= state.dataWindowAC->NumWindAC; ++windowACIndex) {
+            auto &windowAC = state.dataWindowAC->WindAC(windowACIndex);
+            if (windowAC.OutAirVolFlow == 0) {
+                windowACOutdoorAir = true;
+            } else {
+                windowACOutdoorAir = false;
+            }
+            int FanInletNodeIndex = 0;
+            int FanOutletNodeIndex = 0;
+            FanInletNodeIndex = state.dataFans->fans(windowAC.FanIndex)->inletNodeNum;
+            FanOutletNodeIndex = state.dataFans->fans(windowAC.FanIndex)->outletNodeNum;
+
+            if (windowACOutdoorAir &&
+                (NodeNumber == windowAC.OutsideAirNode || NodeNumber == windowAC.MixedAirNode || NodeNumber == windowAC.AirReliefNode ||
+                 NodeNumber == FanInletNodeIndex || NodeNumber == FanOutletNodeIndex || NodeNumber == windowAC.AirInNode)) {
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
I could nitpick here, 

- `windowACOutdoorAir = windowAC.OutAirVolFlow == 0;`
- Why are you declaring the node indexes equal to zero and immediately reassigning them?
- The logic in the node number IF check is pretty difficult to read, could it be written in a more meaningful/readable way?

None of these are actual requests unless there is other work that needs done.